src/Service.Tests/SqlTests/GraphQLSupportedTypesTests/GraphQLSupportedTypesTestsBase.cs

```diff
@@ -426,8 +461,7 @@ public async Task InsertInvalidTimeIntoTimeTypeColumn(string type, string value)
         [DataRow(BOOLEAN_TYPE, true)]
         [DataRow(DATETIMEOFFSET_TYPE, "1999-01-08 10:23:54+8:00")]
         [DataRow(DATETIME_TYPE, "1999-01-08 10:23:54")]
-        [DataRow(TIME_TYPE, "\"23:59:59.9999999\"")]
-        [DataRow(TIME_TYPE, "null")]
+        [DataRow(TIME_TYPE, "23:59:59")]
```

Comment:
why this change?

Reply:
This test shows as green in main branch. However, looking at the log of the test, I saw this error:
>     GraphQL error: [{"message":"The variable `param` is not compatible with the type of the current location.","locations":[{"line":1,"column":65}],"path":["createSupportedType"],"extensions":{"variable":"param","variableType":"Time","locationType":"LocalTime","specifiedBy":"http://spec.graphql.org/October2021/#sec-All-Variable-Usages-are-Allowed"}}]

There are two layers of changes here:
1. TIME_TYPE is technically not a GraphQL data type recognized or defined by DAB's generated GraphQL schema/ HotChocolate -> LocalTime is. So when the data row for TIME_TYPE comes in this code resolves the expected type:
```csharp
        private static string TypeNameToGraphQLType(string typeName)
        {
            return typeName switch
            {
                DATETIMEOFFSET_TYPE => DATETIME_TYPE,
                TIME_TYPE => LOCALTIME_TYPE,
                _ => typeName
            };
        }
```
2. Leaving quotes on the value -> `\"23:59:59.9999999\"` in results in the error: 
>     GraphQL error: [{"message":"Unable to deserialize string to LocalTime","path":["param"]}]

Reply:
why keep data row as "TIME_TYPE"? well, our test code resolves that type name not only as the GraphQL type but also as a way to determine which column in the data source to select which is named time_types. So we'd need to decouple the GraphQL Data Type value from being used to determine the SQL table column name.

Reply:
added to pr description.

Reply:
makes sense